eegrasp/eegrasp.py

```diff
@@ -73,15 +73,12 @@ def euc_dist(self, pos):
             Dimension of the array is number of channels by number of channels
             containing the euclidean distance between each pair of channels.
         """
-
+        from scipy.spatial import distance_matrix
+        
         distance = np.zeros([pos.shape[0], pos.shape[0]],
                             dtype=np.float64)  # Alocate variable
         pos = pos.astype(float)
-        for dim in range(pos.shape[1]):
```

Comment:
Are these changes related to the PR?

Reply:
This change was aimed to solve the same problem presented in [PR45](https://github.com/gsp-eeg/EEGraSP/pull/45) (which can be discarded if this change is approved), but with fewer lines of code, and with minimal code intervention, the solution was proposed by @julioRodino and discussed in a previous meeting as well, without this change the examples that use distance calculation will not run, and neither will the docs.

Reply:
OK